XmlParser.lua

```diff
@@ -363,19 +363,23 @@ end
 
 local function parseTagType(self, xml, f)
     -- Test for tag type
+    local tag
     if string.find(string.sub(f.tagstr, 1, 5), "?xml%s") then
         tag = parseXmlDeclaration(self, xml, f)
     elseif string.sub(f.tagstr, 1, 1) == "?" then
-        tag = parseXmlProcessingInstructions(self, xml, f)
```

Comment:
The function is parseXmlProcessingInstruction. You can remove the commented lines now.